armi/physics/fuelCycle/tests/test_fuelHandlers.py

```diff
@@ -490,6 +491,13 @@ def test_repeatShuffles(self):
         for a in self.r.sfp.getChildren():
             self.assertEqual(a.getLocation(), "SFP")
 
+        # Do some cleanup, since the fuelHandler Interface has code that gets
```

Comment:
This code is that?

Reply:
not sure what you're asking here.

Reply:
`runShuffling` calls `manageFuel` in the fuel handler interface. This makes the shuffles png's, as well as calls `makeShuffleReport`. Both of these use an `fName` variable that functions as absolute. Putting `test_repeatShuffles` inside a TempDirChanger does not alleviate the files left behind that are manually removed here. 

We could change `manageFuel` and `makeShuffleReport` to take the `fName` arg so the test can give it a full file path, but that's a code change that would require a lot of testing and may have unintended consequences.